src/java/main/org/apache/zookeeper/server/FinalRequestProcessor.java

```diff
@@ -430,6 +432,7 @@ public void processRequest(Request request) {
             // the client and leader disagree on where the client is most
             // recently attached (and therefore invalid SESSION MOVED generated)
             cnxn.sendCloseSession();
+            request.checkLatency();
```

Comment:
close session takes time. Is it intended to put the latency check after closing the session instead of before the session closing?